pkg/webhooks/scc/scc.go

```diff
@@ -59,25 +58,13 @@ var (
 )
 
 type SCCWebHook struct {
-	s runtime.Scheme
+	scheme *runtime.Scheme
 }
 
 // NewWebhook creates the new webhook
 func NewWebhook() *SCCWebHook {
-	scheme := runtime.NewScheme()
-	err := admissionv1.AddToScheme(scheme)
-	if err != nil {
-		log.Error(err, "Fail adding admissionsv1 scheme to SCCWebHook")
-		os.Exit(1)
-	}
-	err = corev1.AddToScheme(scheme)
-	if err != nil {
-		log.Error(err, "Fail adding corev1 scheme to SCCWebHook")
-		os.Exit(1)
-	}
-
```

Comment:
It looks like we're no longer adding `admissionv1` and `corev1` to our scheme. Is that because they're no longer needed in the webhook logic? Or does `runtime.NewScheme()` somehow know to add those by default?

Reply:
TIL :slightly_smiling_face: